script/deploy/safe/confirm-safe-tx.ts

```diff
@@ -237,16 +301,22 @@ const func = async (network: string, privateKey: string, rpcUrl?: string) => {
     }
 
     if (action === 'Sign') {
-      await signTransaction(txToConfirm)
+      try {
+        await signTransaction(txToConfirm, safeWebUrl)
+      } catch {}
     }
 
     if (action === 'Sign & Execute Now') {
-      await signTransaction(txToConfirm)
-      await executeTransaction(txToConfirm)
+      try {
+        await signTransaction(txToConfirm, safeWebUrl)
+        await executeTransaction(txToConfirm, safeWebUrl)
+      } catch {}
```

Comment:
Should maybe at least log the error so we know what happened?

Reply:
it does log it in the functions itself